src/openrct2/actions/StaffSetColourAction.hpp

```diff
@@ -44,12 +44,14 @@ DEFINE_GAME_ACTION(StaffSetColourAction, GAME_COMMAND_SET_STAFF_COLOUR, GameActi
     void Serialise(DataSerialiser & stream) override
     {
         GameAction::Serialise(stream);
+
         stream << DS_TAG(_staffType) << DS_TAG(_colour);
     }
 
     GameActionResult::Ptr Query() const override
     {
-        if (_staffType != STAFF_TYPE_HANDYMAN && _staffType != STAFF_TYPE_MECHANIC && _staffType != STAFF_TYPE_SECURITY)
+        if (_staffType != static_cast<uint8_t>(StaffType::Handyman) && _staffType != static_cast<uint8_t>(StaffType::Mechanic)
+            && _staffType != static_cast<uint8_t>(StaffType::Security))
```

Comment:
Maybe this looks more readable as:
```suggestion
        auto staffType = static_cast<StaffType>(_staffType);
        if (staffType != StaffType::Handyman && staffType != StaffType::Mechanic && staffType != StaffType::Security)
```